### PR TITLE
feat(congestion) model of the strategy proposed in NEP

### PR DESCRIPTION
**Pull request by wacban**
This is the strategy as proposed in the NEP.

### CODE REVIEW

File: tools/congestion-model/src/main.rs [hunk 1]
```diff
@@ -161,9 +167,10 @@ fn strategy(strategy_name: &str, num_shards: usize) -> Vec<Box<dyn CongestionStr
             "No queues" => Box::new(NoQueueShard {}) as Box<dyn CongestionStrategy>,
             "Global TX stop" => Box::<GlobalTxStopShard>::default(),
             "Simple backpressure" => Box::<SimpleBackpressure>::default(),
-            "Fancy Global Transaction Stop" => Box::<FancyGlobalTransactionStop>::default(),
+            "Fancy Stop" => Box::<NepStrategy>::default(),
```
Comment (jakmeier) [inline tools/congestion-model/src/main.rs:170]: Is this intentionally mapping to NepStrategy?

File: tools/congestion-model/src/strategy/mod.rs [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 pub use global_tx_stop::GlobalTxStopShard;
+pub use nep::NepStrategy;
 pub use new_tx_last::NewTxLast;
 pub use no_queues::NoQueueShard;
 pub use simple_backpressure::SimpleBackpressure;
 pub use traffic_light::TrafficLight;
 
-mod fancy_global_transaction_stop;
 mod global_tx_stop;
+mod nep;
```
Comment (jakmeier) [inline tools/congestion-model/src/strategy/mod.rs:12]: Are you trying to delete `fancy_global_transaction_stop` or was that a mistake? I think you left the file in but removed the link to it. Probably it should stay in and stay properly linked, right=?

**Review by jakmeier — APPROVED** [review]
The strategy looks good, it's just unclear to me why you removed the fancy global stop strategy. Perhaps an oversight?

File: tools/congestion-model/src/main.rs (hunk 1 above)
Comment (wacban) [reply tools/congestion-model/src/main.rs:170]: Oh no good catch thanks! I was copy pasting files and renaming things and this sneaked in.

**Pull request merged**